docs/DeveloperGuide.md

```diff
@@ -116,7 +128,7 @@ The Class Diagram for the Logic Component is shown below (*Figure 5*)
 
 Given below is the Sequence Diagram (*Figure 6*) for interactions within the `Logic` component for the `execute("delete 1")` API call.
 
-<p align="center"><img src="images/DeleteSequenceDiagram.png"/></p>
+<p align="center"><img src="images/DeleteS:qaequenceDiagram.png"/></p>
```

Comment:
Is this a typo? The image does not seem to load in the preview page.

Reply:
Resolved typo